test/e2e/benchmark/benchmark.go

```diff
@@ -20,14 +25,30 @@ type BenchmarkTest struct {
 }
 
 func NewBenchmarkTest(name string, manifest *Manifest) (*BenchmarkTest, error) {
-	// create a new testnet
-	testNet, err := testnet.New(context.Background(), name, seed,
-		testnet.GetGrafanaInfoFromEnvVar(), manifest.ChainID,
-		manifest.GetGenesisModifiers()...)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	identifier := fmt.Sprintf("%s_%s", name, time.Now().Format(timeFormat))
+	kn, err := knuu.New(ctx, knuu.Options{
+		Scope:        identifier,
+		ProxyEnabled: true,
+	})
 	if err != nil {
 		return nil, err
 	}
 
+	// context.Background() is used to allow the stopSignal to be functioning even after this function returns
```

Comment:
```suggestion
	// context.Background() is used to allow the stopSignal to be functional even after this function returns
```

Reply:
[question] the other tests use `ctx` instead of `context.Background()` here. Since ctx is:

```
	ctx, cancel := context.WithCancel(context.Background())
```

that seems possible to do here too.

Reply:
In other tests the ctx remains alive until the end of the test, so it is ok to use it. In the benchmark, this function does some initialization and ends so the ctx dies and to handle Stop signal we need a ctx that is alive until the end of the test/program. that's why `context.Background()` is used.